src/app/modules/angular-slickgrid/models/formatter.interface.ts

```diff
@@ -1,3 +1,3 @@
 import { Column } from './column.interface';
 
-export type Formatter = (row: number, cell: number, value: any, columnDef: Column, dataContext: any, grid?: any) => string;
+export type Formatter = (row: number, cell: number, value: any, columnDef: Column, dataContext: any, grid?: any) => any;
```

Comment:
I cannot accept `any` Type, it's way too generic and will open a "can of worm".  You have provide me detail of what you said and also provide a sample code of what you are trying to achieve